docs/content/en/blog/news/may-2022-update.md

```diff
@@ -0,0 +1,84 @@
+---
+date: 2022-06-01
+title: "May 2022 update"
+linkTitle: "May 2022 update"
+weight: 993
+description: "Development status update to recap what happened last few months"
+author: Khanh Tran ([@khanhtc1202](https://twitter.com/khanhtc1202))
+---
+
+_Published by the PipeCD dev team every few months, this update will provide you with news and updates about the project! Please click [here](/blog/2022/02/10/february-2022-update/) if you want to see the last status update._
+
+### What's changed
+---
+
+It's been a while from last update about the development of our PipeCD project, but you can be assured, our PipeCD development is going well. Since the last report, PipeCD team has introduced 5 major release (from [v0.27.0](#) to the latest [v0.32.3](#)) with serveral UI improvements and some interesting features that I can wait to show you now. This blog post walks you through some of notable changes. For all other changes, please check out each release note.
+
+#### Cloud Run live state
+
+Until now, you may notice that only applications of kind Kubernetes have a part in its application detail page named live-state which show you the situation of components of the current running PipeCD application. From [version v0.27.0](https://github.com/pipe-cd/pipecd/releases/tag/v0.27.0) we made that feature available for applications of kind Cloud Run too.
+
+![](/images/cloudrun-live-state.png)
+
+#### Make analysis stage skippable
+
+One of the outstanding features PipeCD has is [Automated deployment analysis (ADA)](/docs/user-guide/automated-deployment-analysis/) which helps you evaluate the impact of the current deployment right middle its running. Though it’s necessary to have that kind of stage for the stability of your service, sometimes you may want to skip the stage for some kind of quick/hot fixes. In that case, this feature is for you.
+
+![](/images/analysis-skippable.png)
+
+#### Enable to load Helm chart from OCI registry
+
+With the release of Helm [version v3.8.0](https://helm.sh/blog/storing-charts-in-oci/), Helm is able to store and work with charts in container registries, as an alternative to Helm repositories. PipeCD adopts that feature right the way and for now, you're able to fetch Helm chart from wherever OCI registries and use it to deploy your services. You can find more about how to use this feature in detail in this [docs](/docs/operator-manual/piped/adding-helm-chart-repository-or-registry/#adding-helm-chart-registry).
+
+#### The very first touchable FileDB
+
+It’s been a while since the first time we made a [tweet](https://twitter.com/nghialv2607/status/1480712569535209472) about how PipeCD aims to remove the database as its dependencies to make the installation more easier and compact, today we want to make the first achievement on the road to #nodatabase world.  
```

Comment:
```suggestion
It’s been a while since the first time we made a [tweet](https://twitter.com/nghialv2607/status/1480712569535209472) about how PipeCD aims to remove the database as its dependencies to make the installation easier and compact, today we want to make the first achievement on the road to #nodatabase world.
```